docs/index.rst

```diff
@@ -8,6 +8,9 @@ Pupil Invisible device to the `labstreaminglayer <https://github.com/sccn/labstr
 Install and Usage
 ==================
 
+Make sure the version of your Pupil Invisible Companion App is at least v1.4.14 or higher. You can download
+the latest version of the App in the Play Store on your Pupil Invisible Companion device.
```

Comment:
I think this is a good place to use an ["Admonition"](https://pradyunsg.me/furo/reference/admonitions/). Maybe the `.. important::` one? It might also make sense to move it further down, next to the `.. caution::` one. @ClaraKuper What do you think?

Reply:
How about a custom title "troubleshooting", with the style of "hint" - question mark on green ground

Reply:
Sounds great! Go please ahead with this implementation.